index.html

```diff
@@ -122,6 +122,37 @@ <h2>A project for first-time contributions</h2>
         <!-- ============== ^^^^ TEMPLATE ^^^^ ============== -->
         <!-- ============================================================== -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+        
+        <!-- ________ *rikhils* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">rikhils</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/rikhils" target="_blank">@rikhils</a>
+          </p>
+          <p class="about">Computer Science student at Georgia Tech</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">
```

Comment:
kindly attach the link of the desired website in place of `<a href = #` replace the `#` with your link.